frontend/src/views/SequenceEditView.vue

```diff
@@ -64,9 +72,11 @@
           :key="selectedSlideIndex"
           :slide="selectedSlide"
           :edit-mode="true"
+          :editing="currentEditingSlot ?? undefined"
           class="h-full absolute"
           @editing="
             (val) => {
+              addUnloadEventListener();
```

Comment:
Shouldn't stuff like changing the slide background color, title of sequence, adding slides, etc. also add the unload listener?

Reply:
@ProgramPhoenix

Reply:
review again should now be working